Integrations/integration-AbuseDB.yml

```diff
@@ -161,26 +162,31 @@ script:
         if not isinstance(info, list):
             info = [info]
 
-        context, table, dbot_scores = [], [], []
+        context_ip_generic, context_ip, human_readable, dbot_scores = [], [], [], []
         for analysis in info:
-            analysis_info = {
+            ip_ec = {
+                "Address" : analysis.get("ipAddress"),
+                "Geo" : { "Country" : analysis.get("countryName") or analysis.get("countryCode") }
```

Comment:
what happens if analysis don't have `countryName` or `countryCode`? we get `None`?

Reply:
createContext(categories, *removeNull=True*) should ignore it, but i will check.

Reply:
ok it's showing as "null". I'll talk to you about it tomorrow.